celery_progress/websockets/backend.py

```diff
@@ -1,39 +1,32 @@
-import logging
-
 from celery_progress.backend import ProgressRecorder, Progress
 
 try:
     from asgiref.sync import async_to_sync
     from channels.layers import get_channel_layer
 except ImportError:
-    async_to_sync = get_channel_layer = None
-    WEBSOCKETS_AVAILABLE = False
+    channel_layer = None
 else:
-    WEBSOCKETS_AVAILABLE = get_channel_layer()
-
+    channel_layer = get_channel_layer()
 
-logger = logging.getLogger(__name__)
+if not channel_layer:
+    RuntimeError(
+        'Tried to use websocket progress bar, but dependencies were not installed / configured. '
+        'Use pip install celery-progress[websockets] and set up channels to enable this feature. '
+        'See: https://channels.readthedocs.io/en/latest/ for more details.'
+    )
```

Comment:
This means that anything that ever imports from this file will raise an exception here if things aren't configured, yeah?

I'm not sure that's a problem, but it feels a bit less risky if the error gets raised when you actually try to use the class, not just import it. Is there a reason you didn't do it that way?

Reply:
This will error if the default channel layer has not been configured. I originally thought it would be best to catch the error before it happens, and only evaluate whether the channel layer exists once vs doing it every time a task is updated, but I understand if the other change would make more sense. Although, I now notice that that error will do nothing anyways, as it's not raised at all. How I managed to remove raise during testing and not put it back before committing I will never know. I will push a new commit moving it back into `WebSocketProgressRecorder`.

Reply:
Would you rather it stay as a logging error, or as a RuntimeError?